src/nnet/nnet-blstm-projected-streams.h

```diff
@@ -19,8 +19,8 @@
 
 
 
-#ifndef KALDI_NNET_NNET_BLSTM_PROJECTED_STREAMS_H_
-#define KALDI_NNET_NNET_BLSTM_PROJECTED_STREAMS_H_
+#ifndef KALDI_NNET_BLSTM_PROJECTED_STREAMS_H_
```

Comment:
BTW, please try to get the whitespace correct.  Run your code through
cpplint.py to check.
Here:
nrecur_(int32(output_dim/2)),
there are two issues: for casting you should static_cast<int32> instead of
a C-style cast.
And, like many other places, there is not the recommended whitespace around
operators; should be
output_dim / 2.
I'm not really checking the code for content; I assume Karel and others
will do that.
Dan


On Thu, Aug 13, 2015 at 6:54 PM, Kirill Katsnelson <notifications@github.com
> wrote:

> In src/nnet/nnet-blstm-projected-streams.h
> <https://github.com/kaldi-asr/kaldi/pull/66#discussion_r37044335>:
>
> > @@ -19,8 +19,8 @@
> >
> >
> >
> > -#ifndef KALDI_NNET_NNET_BLSTM_PROJECTED_STREAMS_H_
> > -#define KALDI_NNET_NNET_BLSTM_PROJECTED_STREAMS_H_
> > +#ifndef KALDI_NNET_BLSTM_PROJECTED_STREAMS_H_
>
> NNET_NNET is not a mistake: the file is referred to as
> "nnet/nnet-blstm-projected-streams.h". Please revert these guard names.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/kaldi-asr/kaldi/pull/66/files#r37044335>.
>